# marauders: name=unescape_predef_entities
diff --git a/src/events/attributes.rs b/src/events/attributes.rs
--- a/src/events/attributes.rs
+++ b/src/events/attributes.rs
@@ -83,7 +83,7 @@
     #[cfg(any(doc, not(feature = "encoding")))]
     pub fn normalized_value(&self, version: XmlVersion) -> XmlResult<Cow<'a, str>> {
         // resolve_predefined_entity returns only non-recursive replacements, so depth=1 is enough
-        self.normalized_value_with(version, 1, resolve_predefined_entity)
+        self.normalized_value_with(version, 1, |_| None)
     }
 
     /// Returns the attribute value normalized as per [the XML specification] (or [for 1.0]),
